zingolib/src/lightclient.rs

```diff
@@ -1162,6 +1162,16 @@ impl LightClient {
         *self.interrupt_sync.read().await
     }
 
+    pub async fn get_wallet_kind(&self) -> Result<JsonValue, &str> {
```

Comment:
If I read only the name of the fn in calling code..  then I will get really confused about the return types.

How about we define a two-variant enum:

```Rust
enum WalletKind {
       Spend,
       View,
}
```

And we have `get_wallet_kind` return a `WalletKind`?